packages.json

```diff
@@ -933,6 +933,19 @@
     "web": "https://github.com/PMunch/nim-pcap"
   },
   {
+    "name": "drawille",
+    "url": "https://github.com/PMunch/drawille-nim",
+    "method": "git",
+    "tags": [
+      "drawile",
+      "terminal",
+			"graphics"
```

Comment:
what happened here? :)

Reply:
Ugh, my Vim is a bit too happy with tabs..